### DISCUSSION TITLE
Support needed for v0.16.45

### DISCUSSION THREAD

**Discussion by aravind-111** [GrapesJS/grapesjs#5216] [General]
Hello,

I am using Grapesjs version 0.16.45 and the following is my storage manager

Storage manager

```
const storageManager = useMemo(() => ({
        type: "remote",
        fromElement: false,
        id: "gjs-",
        contentTypeJson: true,
        urlStore: storeTemplate,
        headers: { ... } ,
    }), []);
```

See I run my react app in 3000 and server app in 8080 when `type: 'remote'` a request is sent to the current client URL.

In my case, a GET request is sent to `http://localhost:3000/editor/edit/template/Ynfqkl6HocI9Ju9WY7i6oyc99l0ki` (which is my current URL where I am using editor)  and the response is `304`

Other than this everything is working fine. My `urlStore` is getting called as expected and the template is updated in the DB.

What is the issue here?

**Comment by artf** [comment]
This is not related to GrapesJS itself, looks like you just need to pass to `urlStore ` the path with your base URL (eg. `http://localhost:8080/...`) to hit the server app instead of the client dev.

**Reply by aravind-111** [reply]
In the above storage manager, I provided `urlStore: storeTemplate`,  `storeTemplate` variable is the server's URL.
